### DISCUSSION TITLE
ALVR OpenXR Raspberry PI

### DISCUSSION THREAD

**Discussion by raeldor** [alvr-org/ALVR#985] [General]
Hi.  I'm interested in building a homebrew VR headset.  Is it possible to run ALVR on a Raspberry PI 4 running Android (or any other OS) using OpenXR?  Thanks.

**Comment by zmerp** [comment]
ALVR currently does not support OpenXR. I'm working on the necessary changes to support OpenXR but it will be a while until support is completed. That being said, there is a [fork](https://github.com/korejan/ALVR) of ALVR that implements OpenXR support. Make sure the Raspberry Pi 4 supports MediaCodec hardware acceleration for h264.

**Comment by raeldor** [comment]
I appreciate the link, thank you.  Will you be working on your implementation separately, or merging the fork?  Thanks!

**Reply by zmerp** [reply]
That fork is working separately and will not be merged into the main repository. There is already an implementation in progress in this repository but it still requires quite a bit of work to make it functional.

**Comment by iowesty** [comment]
> Hi. I'm interested in building a homebrew VR headset. Is it possible to run ALVR on a Raspberry PI 4 running Android (or any other OS) using OpenXR? Thanks.

were u able to get any where with the alvr usage on the pi4, looking to try it myself but with pi5 and bigscreen2?
